Form-Controls/index.html

```diff
@@ -9,19 +9,53 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
   <header>
-    <h1>Product Pick</h1>
+    <h1>T-shirt order form</h1>
+    <h2 id="heading">choose your t-shirt here!</h2>
   </header>
+
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+
+  <form method="post" class="form">
+      <label for="name" ><h2>Name:</h2></label>
+      <input type="text" id="name" name="user_name" minlength="2"  maxlength="20" placeholder="Enter your name here" required/>
+      <label for="mail"><h2>Email:</h2></label>
+      <input type="email" id="mail" name="user_email" placeholder="example@example.co.uk" pattern="[^@\s]+@[^@\s]+\.[^@\s]+" title="Invalid email address" required/>
+    <div>
+      <h2>Choose T-shirt color:</h2>
+      <div  id="color" >
```

Comment:
Is there a grouping element for forms that could be useful here? Could this tutorial help you answer this question:

https://www.w3.org/WAI/tutorials/forms/grouping/#radio-buttons